apt/private/deb_import.bzl

```diff
@@ -9,7 +9,10 @@ _RECOMPRESS_CMD = "$(ZSTD_BIN) -f --decompress $< --stdout | $(ZSTD_BIN) - --for
 genrule(
     name = "data",
     srcs = glob(["data.tar.*"]),
-    outs = ["data.tar.gz"],
+    # recompress to a different output name (i.e. not 'data.tar.gz')
+    # because in the case of no extension, we would have data.tar.gz
+    # in both input and output.
+    outs = ["out.tar.gz"], 
```

Comment:
This is a good find honestly, for other bits, we have a fix in place; but if you'd like to clean up this PR, i would like to land this fix right here; 

See https://github.com/GoogleContainerTools/rules_distroless/pull/129 for other bits.

Reply:
sure will target EOD PST hopefully

Reply:
I believe I already fixed this in 35a7d5a3

Reply:
Nice, feel free to close this one if it's totally superseded